src/main/java/org/rutebanken/irkalla/domain/CrudEvent.java

```diff
@@ -132,7 +138,10 @@ public Builder location(String location) {
 
 
         public Builder newValue(String newValue) {
-            event.newValue = newValue;
+            if (newValue !=null && newValue.length() > MAX_STRING_LENGTH) {
+                logger.warn("Trimming newValue since its greater than max string length: " + newValue);
```

Comment:
Use placeholders in log messages:
"Trimming newValue since its greater than max string length : {} ", newValue)